include/DREAM/Equations/Fluid/IonEquationTerm.hpp

```diff
@@ -19,6 +19,7 @@ namespace DREAM {
 
     public:
         IonEquationTerm(FVM::Grid*, IonHandler*, const len_t iIon);
+        IonEquationTerm(FVM::Grid*, IonHandler*, const len_t iIon, bool allocCoefficients);
```

Comment:
Remove the other constructor and change the last parameter here to ``bool allocCoefficients=false``?